### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akiezun**
add link to gatk-protected

@vdauwera for you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 ###GATK 4 (codename Hellbender)
 
-The public parts of the next generation of GATK/Picard methods engine and tools.
+This repository contains the public parts of the next generation of GATK/Picard methods engine and tools). See also [gatk-protected](https://github.com/broadinstitute/gatk-protected) for the parts of the toolkit that are licence-protected.
```
Comment (vdauwera) [inline README.md]: license with an 's' in US english

(http://grammarist.com/spelling/licence-license/)

Comment (vdauwera) [inline README.md]: I might rephrase this as something like "protected by a license that restricts their use to non-commercial applications". 

We should probably add a note at some point explaining the two-repo structure a bit more clearly. Feel free to assign an issue to me to do that.

**Comment by vdauwera** [comment]
See my minor correction/suggestion. Should follow up with a task to explain the two-repo structure clearly.

**Comment by akiezun** [comment]
updated. let me know if this is better

**Comment by akiezun** [comment]
the 2-repo ticket is https://github.com/broadinstitute/gatk/issues/1208

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 ###GATK 4 (codename Hellbender)
 
-The public parts of the next generation of GATK/Picard methods engine and tools.
+This repository contains the next generation of GATK/Picard engine and the freely available tools (see [LICENSE](https://github.com/broadinstitute/gatk/blob/master/LICENSE.TXT)). See also [gatk-protected ](https://github.com/broadinstitute/gatk-protected) repository for the GATK tools that are restricted to use only in non-commercial applications.
```
Comment (vdauwera) [inline README.md]: much better, but add (please contact softwarelicensing@broadinstitute.org to inquire about licensing for commercial applications) otherwise people may not realize it's possible to use the tools commercially if you pay for it.

Comment (lbergelson) [inline README.md]: I think putting a big paragraph about restrictive licensing as the first thing people see on our page is maybe not the best thing.

Comment (vdauwera) [inline README.md]: OK so how about replacing 

    See also [gatk-protected ](https://github.com/broadinstitute/gatk-protected) repository for the GATK tools that are restricted to use only in non-commercial applications.

with 

     See also [gatk-protected ](https://github.com/broadinstitute/gatk-protected) repository for additional GATK tools that are distributed under a different license.

Comment (lbergelson) [inline README.md]: :+1:

**Comment by akiezun** [comment]
is this mine now or still with @vdauwera ?

**Comment by vdauwera** [comment]
All yours, sorry
